src/components/Alteration/Articles/CompanyProvisions.vue

```diff
@@ -54,7 +54,7 @@
                 >
                   <v-list-item-subtitle>
                     <v-icon small color="primary">mdi-pencil</v-icon>
-                    <span class="drop-down-action ml-1">{{ editLabel }}</span>
+                    <span class="drop-down-action ml-1">Change</span>
```

Comment:
Note that dropdown button remains as "Change" (since we're changing an edit)... confirmed with Yui.